src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2411,6 +2411,16 @@ const bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType
     return false;
   }
   if (UseSVE > 0) {
+    switch (opcode) {
+      case Op_VectorLoadShuffle:
+      case Op_VectorRearrange:
+        if (vlen < 4) {
+          return false;
+        }
+        break;
+      default:
+        break;
+    }
```

Comment:
I would prefer put this inside op_sve_supported.

Reply:
Yes, I will fix that.